doc/sphinx/practical-tools/utilities.rst

```diff
@@ -169,6 +169,24 @@ You can list multiple directories if you wish.
    a.v
    b.v
 
+If your project is made up of multiple sub-projects, you can specifically use the `-L`
```

Comment:
This section is introducing basic concepts.  -L is not basic, it should be described later, perhaps in [this section](http://localhost:63342/coq2/_build/default/doc/refman-html/practical-tools/utilities.html#modifying-multiple-interdependent-projects-at-the-same-time)

IIUC you would not use -L to refer to other packages that you're not modifying as part of your project.